Forms/MainWindow.ui

```diff
@@ -2188,6 +2188,9 @@
      <height>22</height>
     </rect>
    </property>
+   <property name="nativeMenuBar">
+    <bool>false</bool>
+   </property>
```

Comment:
Nice, didn't know that.
What is the OS the menu bar is broken in?

Reply:
My example specifically is on Arch Linux; though this is likely tied to the rolling release of kwin/kde plasma (practically a nonissue on the kwin/plasma builds on fixed release distros; this issue also just appeared last week on my main system, and isn't present on my laptop (arch based) with an older version of plasma & kwin)